Modules/Microsoft365DSC/DSCResources/MSFT_IntuneAppConfigurationPolicy/MSFT_IntuneAppConfigurationPolicy.psm1

```diff
@@ -634,6 +623,7 @@ function ConvertTo-M365DSCIntuneAppConfigurationPolicyCustomSettings
         $result += $currentSetting
     }
     return $result
+    Remove-M365DSCAuthenticationParameter
```

Comment:
This one was most likely a copy pasto, please remove it :)

Reply:
Good catch - not sure where that came from ...